src/utils/dom/renderers/renderInput.js

```diff
@@ -3,36 +3,45 @@ import { warn, error, isPromise } from '../../utils.js'
 import * as dom from '../../dom/index.js'
 import privateProps from '../../../privateProps.js'
 
+const inputTypes = ['input', 'file', 'range', 'select', 'radio', 'checkbox', 'textarea']
+
 export const renderInput = (instance, params) => {
+  const content = dom.getContent()
   const innerParams = privateProps.innerParams.get(instance)
   const rerender = !innerParams || params.input !== innerParams.input
-  const content = dom.getContent()
-  const inputTypes = ['input', 'file', 'range', 'select', 'radio', 'checkbox', 'textarea']
-  for (let i = 0; i < inputTypes.length; i++) {
-    const inputClass = swalClasses[inputTypes[i]]
-    const inputContainer = dom.getChildByClass(content, inputClass)
+  const hasInputOnParams = params.input != undefined
```

Comment:
unnecessary variable, checking `!params.input` will work the same job

Reply:
nice. Removed.